content/applications/finance/accounting/getting_started/process_overview/supplier_bill.rst

```diff
@@ -1,144 +1,106 @@
-prices.
+Bills creation
+==============
 
-.. image:: supplier_bill/vendor_bill01.png
-   :align: center
+From the Accounting app, vendor bills can either be created manually or automatically.
 
-Save the invoice to update the pre tax and tax amounts at the bottom of
-the screen. You will most likely need to configure the prices of your
-products without taxes as Odoo will compute the tax for you.
-
-.. note::
-    On the bottom left corner, Odoo shows a summary table of all taxes on the vendor bill.
-    In several countries, different methods are accepted to round the totals (round per line,
-    or round globally). The default rounding method in Odoo is to round the final prices
-    per line (as you may have different taxes per product. E.g. Alcohol and cigarettes).
-    However if your vendor has a different tax amount on their bill, you can change the
-    amount in the bottom left table to adjust and match.
+Manually
+--------
 
-Validate The Vendor Bill
-------------------------
+It is possible to manually create a vendor bill by going to :menuselection:`Vendors --> Bills` and
+then clicking :guilabel:`Create`.
 
-Once the vendor bill is validated, a journal entry will be generated
-based on the configuration on the invoice. This journal entry may differ
-depending on the the accounting package you choose to use.
+Automatically
+-------------
 
-For most European countries, the journal entry will use the following
-accounts:
+Vendor bills can be automatically created by **sending an email** to an :ref:`email alias
+<invoice-digitization/email-alias>` associated with the purchase journal, or by **uploading a pdf**
+in :menuselection:`Accounting app --> Vendors --> Bills` and then clicking :guilabel:`Upload`.
 
--  **Accounts Payable:** defined on the vendor form
+Bills completion
+================
 
--  **Taxes:** defined on the products and per line
+Whether you create the bill manually or if it is generated automatically, make sure the following
+fields are completed as appropriate:
 
--  **Expenses:** defined on the line item product used
+First, select the :guilabel:`Vendor`. Odoo automatically fills some fields based on previous PO or
+bills from the vendor.
 
-For Anglo-Saxon (US) accounting, the journal entry will use the
-following accounts:
+Record the order reference number in the :guilabel:`Bill Reference` field. This will enables to
+later :ref:`match <payments-matching>` the PO with the vendor bill (as the vendor bill should
+include the Vendor Reference).
 
--  **Accounts Payable:** defined on the vendor form
+The :guilabel:`Auto-Complete` field allows to complete the document from a past bill/purchase order.
+The :guilabel:`Vendor` field should be completed prior to completing this field.
 
--  **Taxes:** defined on the products and per line
+The :guilabel:`Bill Date` is the document date, and the :guilabel:`Accounting Date` is the documennt
+creation date in your accounting.
```

Comment:
```suggestion
creation date in your accounting.
```
I think this isn't clear enough. Sorry, I know you had issues with this, but IMO not clear enough as it right now.  A quick search suggests this, perhaps it would be worth it to look into it: 
![definition](https://user-images.githubusercontent.com/104567387/209686995-9b8d2df0-7e89-46b4-baea-f2514212c463.png)